Source/WPF/MyMoney/Database/Money.cs

```diff
@@ -12718,6 +12718,27 @@ public string InvestmentSecuritySymbol
             }
         }
 
+
+        [XmlIgnore]
+        public string SortableInvestmentSecurity
+        {
+            get
+            {
+                if (this.Investment != null && this.Investment.Security != null)
+                {
+                    if (String.IsNullOrEmpty(this.Investment.Security.Name))

```

Comment:
Looks good, just an FYI nit pick this can now be written `this.Investment?.Security?.Name`

Reply:
I'm not a big fan of this new movement of using short-circuit null check for the following reasons
This is easier to read, deterministic and can be debug with a single breakpoint
```
if(foo!=null){
   bar.a = foo.a
   bar.b = foo.b
   bar.c = foo.c
}
```

this is not as easy to read of understand what happens to the bar values, I had to lookup each language to know if bar.a will be set to null or untouched

```
bar.a = foo?.a
bar.b = foo?.b
bar.c = foo?.c
```

Reply:
I agree in the grouped case where you are going to access a lot of properties on foo, but I do find it convenient when I'm accessing one deep property, since it eliminates a lot of nested if statements.